### DISCUSSION TITLE
I'm need help getting started.

### DISCUSSION THREAD

**Discussion by darkvalleyrecords** [codewars/codewars.com#3048] [General]
I know HTML and GML. Made my own websites and currently designing my own game. I was recommended Codewars to help me expand into more areas. But I don't understand the questions being asked or how to answer them. I have been trying the easiest ones but I don't know what to do.
Are there ant tutorials that can get the ball rolling for me?

**Comment by ChairmanMawh** [comment]
You don't appear to know programming languages; HTML and GML are markup languages and can't really be used to solve the challenges on codewars. You can't learn math by taking math tests; you need to pick a programming language you want to learn and follow some tutorials for that

**Comment by Kacarott** [comment]
When I learn new languages, , the flow I like to do is:
1. Look at a few basic kata in that language and understand the task, even though I don't know how to solve yet
2. Find tutorials/read documentation to learn the basics of the language
3. As soon as I think "I probably know enough to solve one of those kata" I go back and try to solve it. If I cant, then I keep learning until I can.
4. Repeat from step 1.
